requirements.txt

```diff
@@ -1,2 +1,2 @@
-scikit-learn
-pandas
\ No newline at end of file
+scikit-learn==1.5.2
+pandas==2.2.3
```

Comment:
I think for now, let's just lock the version of scikit, for pandas we can let it without a locked version so it can adapt to the other library pandas if needed.